File: src/main/java/com/example/ssauc/user/cash/dto/CalculateDto.java
package com.example.ssauc.user.cash.dto;

import java.time.LocalDateTime;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class CalculateDto {
    private Long orderId;
    private Long paymentAmount;
    private Long productId;
    private String productName;     // orders에서 가져온 상품 이름
    private String productImageUrl;
    private LocalDateTime paymentTime;  // 판매자: orders.completedDate, 구매자: payment.paymentDate
    private String orderStatus;     // orders.order_status
}

File: src/main/java/com/example/ssauc/user/cash/dto/CalculateDto.kt
package com.example.ssauc.user.cash.dto

import lombok.*
import java.time.LocalDateTime

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
class CalculateDto {
    private val orderId: Long? = null
    private val paymentAmount: Long? = null
    private val productId: Long? = null
    private val productName: String? = null // orders에서 가져온 상품 이름
    private val productImageUrl: String? = null
    private val paymentTime: LocalDateTime? = null // 판매자: orders.completedDate, 구매자: payment.paymentDate
    private val orderStatus: String? = null // orders.order_status
}
